src/lambda/commands/importLambda.ts

```diff
@@ -25,49 +25,30 @@ import { LambdaFunctionNode } from '../explorer/lambdaFunctionNode'
 import { getFamily, RuntimeFamily } from '../models/samLambdaRuntime'
 import { showConfirmationMessage } from '../../s3/util/messages'
 import { addWorkspaceFolder } from '../../shared/utilities/workspaceUtils'
+import { promptUserForLocation, WizardContext } from '../../shared/wizards/multiStepWizard'
 
 const pipeline = promisify(Stream.pipeline)
 
 export async function importLambdaCommand(functionNode: LambdaFunctionNode, window = Window.vscode()) {
     const workspaceFolders = vscode.workspace.workspaceFolders || []
-    const labelToWorkspace = new Map(workspaceFolders.map(folder => [`$(root-folder-opened) ${folder.name}`, folder]))
-    const otherLocationName = `$(folder-opened) ${localize('AWS.lambda.import.otherLocation', 'Select a folder...')}`
-
-    // TODO: Move to FolderQuickPickItem (would have to modify BrowseFolderQuickPickItem to non-Wizard dependence)?
-    const selectedLocation = await vscode.window.showQuickPick([...labelToWorkspace.keys(), otherLocationName], {
-        placeHolder: localize('AWS.lambda.import.prompt.placeholder', 'Select the import location'),
-    })
-
-    if (!selectedLocation) {
-        getLogger().info('ImportLambda cancelled')
+    if (workspaceFolders.length === 0) {
+        window.showErrorMessage(
+            localize(
+                'AWS.lambda.import.noWorkspaceFolders',
+                'Please add a workspace folder before importing a Lambda function.'
```

Comment:
"Please" is never needed in messages, better to draw attention to the actual content of the message. User won't be offended :)

    'Select a workspace before importing a Lambda function."